docs/source/WritingTests.rst

```diff
@@ -761,144 +761,71 @@ Setting a Test Timeout
 Sometimes your test suite/test might get stuck forever, and this might
 impact your test grid. You can account for that possibility and set up a
 ``timeout`` parameter for your test. The test timeout can be set through
-2 means, in the following order of precedence:
-
-* Multiplex variable parameters. You may just set the timeout parameter, like
-  in the following simplistic example:
+the multiplex, as shown below. 
```

Comment:
With @ldoktor's work on the parameter system, this will have to be rewritten.  But for now, it's OK.